<ui version="4.0" >
 <class>NoiseStartpointDlg</class>
 <widget class="QDialog" name="NoiseStartpointDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Start point noise</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QRadioButton" name="randomRadio" >
        <property name="text" >
         <string>Random</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="fixedRadio" >
        <property name="text" >
         <string>Fixed point: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="startpointLine" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="minimumSize" >
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="sLbl" >
        <property name="text" >
         <string>s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NoiseStartpointDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NoiseStartpointDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
